src/Component/CodeEditor/CodeEditor.tsx

```diff
@@ -333,81 +247,58 @@ export class CodeEditor extends React.Component<CodeEditorProps, CodeEditorState
     }
   };
 
-  render() {
-    const {
-      locale,
-      showSaveButton,
-      showCopyButton
-    } = this.props;
-    const activeParser: any = this.state.activeParser;
-    const {
-      hasError,
-      value
-    } = this.state;
-    if (hasError) {
-      return <h1>An error occured in the CodeEditor UI.</h1>;
-    }
-
-    return (
-      <div className="gs-code-editor">
-        <div className="gs-code-editor-toolbar" >
-          {locale.formatSelectLabel}: <Select
-            className="gs-code-editor-format-select"
-            style={{ width: 300 }}
-            onSelect={this.onSelect}
-            value={activeParser ? activeParser.title : 'GeoStyler Style'}
-          >
-            {this.getParserOptions()}
-          </Select>
-          {activeParser &&
-          activeParser.sldVersion &&
-          activeParser.sldVersion !== '1.0.0' &&
-            <SLDUnitsSelect
-              changeHandler={this.onUnitSelect}
-            />
-          }
-        </div>
-        <CodeMirror
-          className="gs-code-editor-codemirror"
-          value={value}
-          autoCursor={false}
-          options={{
-            gutters: ['CodeMirror-lint-markers'],
-            lint: true,
-            mode: this.getModeByParser(),
-            lineNumbers: true,
-            lineWrapping: true
-          }}
-          onChange={this.handleOnChange}
-        />
-        <div className="gs-code-editor-errormessage">
-          {this.state.invalidMessage}
-        </div>
-        <div className="gs-code-editor-bottombar">
-          {
-            !showCopyButton ? null :
-              <Button
-                className="gs-code-editor-copy-button"
-                type="primary"
-                onClick={this.onCopyButtonClick}
-              >
-                {locale.copyButtonLabel}
-              </Button>
-          }
-          {
-            !showSaveButton ? null :
-              <Button
-                className="gs-code-editor-download-button"
-                type="primary"
-                onClick={this.onDownloadButtonClick}
-              >
-                {locale.downloadButtonLabel}
-              </Button>
-          }
-        </div>
+  const parserHasUnitSelect = isSldParser && activeParser && (activeParser as SldStyleParser).sldVersion !== '1.0.0';
+
+  return (
+    <div className="gs-code-editor">
+      <div className="gs-code-editor-toolbar" >
+        {locale.formatSelectLabel}: <Select
+          className="gs-code-editor-format-select"
+          style={{ width: 300 }}
```

Comment:
Not related to this PR in particular, but style should be moved to the style definition.